backend/security.py

```
@@ -0,0 +1,184 @@
+                detail="Authentication required",
+                headers={"WWW-Authenticate": "Bearer"}
+            )
+        
+        # Get token
+        token = auth_header.split(" ")[1]
+        
+        # Get user from token
+        try:
+            user = await get_current_user(token)
+            request.state.user = user
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)